07-reputation/04-globaltrust.md

```diff
@@ -0,0 +1,39 @@
+## Global Trust
+
+*Global trust*
+: the result of the [EigenTrust](http://ilpubs.stanford.edu:8090/562/1/2002-56.pdf) algorithm -- the trust in the network participant, which was obtained taking into account *all* local trusts of *all* nodes.
+
+### Subject and Object of a trust
+
+The Subject of the assessment is the entire system. Any node in the network is an Object of trust.
+
+The evaluation criterion is the aggregate of local trust of *all* network participants to the current node, but adjusted for similar local trust to each participant from the rest of the network.
+
+### Calculating trust
+
+Each manager retrieves the local trust value of its child at the end of each epoch and stores it in the `DaughterStorage`. The further task of each manager is to calculate the Global trust of the system in its child node, and also to transfer all the information he has to other managers so that they, in turn, can perform a similar task.
+
+Global trust is calculated iteratively. In the process of calculating `GlobalTrust`, according to the EigenTrust algorithm, managers need to exchange “intermediate” trust values, the so-called `IntermediateTrust`. With the help of these values, managers at each iteration approximate (recalculate the results of the previous iteration) the value of the Global trust to the common Global value limit: $T = [t_0, ..., t_n]$, where $t_i$ is the Global trust value in the node `i` limit.
+
+All formulations, terms and statements in this clause take into account that Global trust makes sense only in the context of a certain epoch, that is, peer-to-peer interactions of nodes for the epoch `n` affect *only* the Global trust calculated in the epoch `n`. `IntermediateTrust` values, likewise, only make sense when used in the context of some epoch **and iteration number**. That fact is not emphasized explicitly further.
```

Comment:
Using all these formulations, terms and statements, we recognize that Local Trust is only applicable in the context of a certain epoch. It means that peer-to-peer interactions of nodes for an epoch `n` affect *only* Local Trusts calculated in the epoch `n`.

Reply:
fixed